keyboards/anavi/macropad8/info.json

```
@@ -0,0 +1,12 @@
+{
+  "keyboard_name": "ANAVI Macro Pad 8",
+  "url": "",
+  "maintainer": "Leon Anavi",
```

This is meant to be your GitHub username, rather than your name.

```suggestion
  "maintainer": "leon-anavi",
```